src/pages/home/report/ReportActionsList.js

```diff
@@ -106,14 +106,14 @@ class ReportActionsList extends React.Component {
     }
 
     /**
-     * Create a unique key for Each Action in the FlatList.
-     * We use the reportActionId that is a string representation of a random 64-bit int, which should be
-     * random enought to avoid colisions
+     * Create a unique key for each action in the FlatList.
+     * We use the reportActionID that is a string representation of a random 64-bit int, which should be
```

Comment:
This comment is only giving a partial explanation. Why is it also using clientID and sequenceNumber? Why isn't the reportActionID enough?

Reply:
Haha, OK. I'm not sure I followed all of that, but I can appreciate if this is in an intermediary state until the full refactoring is complete.